tools/git-changelog.sh

```diff
@@ -8,9 +8,7 @@ tmp="$TMPDIR/changelog"
 
 if [ "$(expr substr $(uname -s) 1 10)" == "MINGW32_NT" ]; then
   NULL="NUL";
-  # TMP was overriden by Bash (TMP="/tmp") we need to unset it.
-  unset TMP;
-  tmp="$TMP/changelog"
+  tmp="._changelog"
```

Comment:
What is the purpose of this change?

I am not entirely against keeping the temp file in the local directory. However, it must be listed in `.gitignore` in such case, to ensure it is not accidentally committed to git and/or released to npmjs.org. The name `.changelog` would be better in such change.